chat/models.py

```diff
@@ -23,6 +35,8 @@ class Conversation(models.Model):
     userEmail = models.CharField(max_length=255)
     title = models.CharField(max_length=255)
     bot = models.ForeignKey(ChatBot, on_delete=models.CASCADE, related_name='conversations')
+    unknownWords = models.ManyToManyField(UnknownWord, related_name='conversations')
```

Comment:
It would be better to handle `'conversations'` as a const for maintainability and readability